app/angular/src/server/angular-cli-webpack-12.2.x.js

```diff
@@ -46,12 +46,9 @@ exports.getWebpackConfig = async (baseConfig, { builderOptions, builderContext }
 
   /**
    * Merge baseConfig Webpack with angular-cli Webpack
+   * Keep all entry from angular cli except main. Add storybook entry to man key
    */
-  const entry = [
-    ...baseConfig.entry,
-    ...(cliConfig.entry.styles ?? []),
-    ...(cliConfig.entry.polyfills ?? []),
-  ];
+  const entry = { ...cliConfig.entry, main: baseConfig.entry };
```

Comment:
Is it correct, that both are called `entry`?

```
{
  ...cliConfig.entry,
  main: baseConfig.entry
}
```

The object looks like this then, right?

```
{
  styles: [...],
  polyfilles: [...],
  main: [...],
}
```

Reply:
Yes. And allow angular to add more entry arrays